server/build/webpack.js

```diff
@@ -194,7 +194,7 @@ export default async function getBaseWebpackConfig (dir, {dev = false, isServer
         next: nextDir,
         // This bypasses React's check for production mode. Since we know it is in production this way.
         // This allows us to exclude React from being uglified. Saving multiple seconds per build.
-        react: dev ? 'react/cjs/react.development.js' : 'eact/cjs/react.production.js',
+        react: dev ? 'react/cjs/react.development.js' : 'react/cjs/react.production.js',
```

Comment:
should it be `react/cjs/react.production.min.js` ?

Reply:
Thanks 🤗